frontend/app/templates/components/checkin/multi-select-group.hbs

```diff
@@ -0,0 +1,22 @@
+{{checkin/multi-select-edit
+  objects=relatedObjects
+  modelName=modelName
+  badgeStyle=badgeStyle
+  selectedTag=selectedObject
+  onClicked=(action "remove")
+  onSelected=(action "add")
+}}
+
+{{multi-group
+  scope="most_popular"
+  title="Popular"
+  modelName=modelName
+  onTagClicked=(action "add")
```

Comment:
I would suggest something more generic like `onItemClicked` rather than `onTagClicked ` as `Tag` has a specific meaning in Flaredown